<launch>
 <rosparam>
  system_mapping:
  -
    pair: [simulator/consumer/manipulator, world_model/simulator/manipulator]

  consumer: simulator/consumer
  </rosparam>


  <rosparam ns="world_model/planning_contexts">
    space1:
     default:
      type: "full_mapping"
  </rosparam>

  <rosparam command="load" ns="world_model" file="$(find prx_input)/templates/simulators/collision_stop_simulator.yaml"/>

  <rosparam command="load" ns="world_model/simulator/subsystems/manipulator" file="$(find baxter)/input/urdf/baxter.yaml"/>
  <rosparam command="load" ns="world_model/simulator/obstacles" file="$(find baxter)/input/simulation/baxter_torso.yaml" /> 

  <rosparam ns="world_model/simulator/">
    simulation_step: 0.02
  </rosparam>


</launch>


